apps/site/lib/site_web/controllers/schedule/line_api.ex

```diff
@@ -0,0 +1,41 @@
+defmodule SiteWeb.ScheduleController.LineApi do
+  @moduledoc """
+    API for retrieving Map data by route and variant
```

Comment:
@meagonqz did you mean "retrieving LineDiagram data"?

Reply:
Copy pasta, thanks